.kwalitee.yml

```diff
@@ -10,3 +10,7 @@ components:
 - tests
 - travis
 - views
+exclude_author_names:
+- Jiri Kuncar <jiri.kuncar@gmail.com>
+- Mateusz Susik <mateusz.susik@cern.ch>
+- Yoan Blanc <yoan.blanc@cern.ch>
```

Comment:
@greut Your canonical email address is well famed out and glorified in the AUTHORS file: :smile: 

```console
$ git grep Yoan AUTHORS.rst 
AUTHORS.rst:* Yoan Blanc <yoan@dosimple.ch>
```